clsCommon.pri

```diff
@@ -14,7 +14,6 @@ HEADERS *=\
 	source/beamline/CLS/CLSValves.h \
 	source/beamline/CLS/CLSShutters.h \
 	source/beamline/CLS/CLSMirrorMaskState.h \
-	source/beamline/CLS/CLSSSRLMonochromatorMaskState.h \
```

Comment:
Yes, I think that's the case. The class isn't used anywhere else, and the rest of the SSRL mono classes are BioXAS-specific. If we'd like to move it up to the CLS level eventually, I'd prefer we move everything up together.